docs/src/pages/guides/migration-v3/migration-v3.md

```diff
@@ -340,3 +340,22 @@ You should be able to move the custom styles to the root class key.
   `event.preventDefault()` is meant to stop default behaviors like clicking a checkbox to check it, hitting a button to submit a form, and hitting left arrow to move the cursor in a text input etc.
   Only special HTML elements have these default behaviors.
   People should use `event.stopPropagation()` if they don't want to trigger a `onClose` event on the modal.
+
+### Tooltip
+
+- [Tooltip] Convert to function component (#15291) @joshwooding
```

Comment:
```suggestion
- [Tooltip] The child needs to be able to hold a ref
```

Reply:
I think that we should remove the pull request and the name of the author in the migration guide. I have forgotten to remove it in the Modal component section.

Reply:
Ahh, I copied the Modal example